flarefly/data_handler.py

```diff
@@ -66,13 +67,13 @@ def __init__(self, data=None, var_name='', limits=None, use_zfit=True, **kwargs)
                             self._nbins_ = len(hist_array[1][idx_min:idx_max])
                             self._limits_[0] = hist_array[1][idx_min]
                             self._limits_[1] = hist_array[1][idx_max]
-                        binning = zfit.binned.RegularBinning(
+                        self._binning_ = zfit.binned.RegularBinning(
                             self._nbins_,
                             self._limits_[0],
                             self._limits_[1],
                             name="xaxis"
                         )
-                        self._obs_ = zfit.Space("xaxis", binning=binning)
+                        self._obs_ = zfit.Space("xaxis", binning=self._binning_)
```

Comment:
I think that there is no need to add the binning as datamember, since you can retrieve the binning from `self._obs_` in the `F2Fitter` class as `self._data_handler_.get_obs().binning`